pkgs/development/tools/continuous-integration/drone-runner-exec/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "drone-runner-exec";
+  version = "2020-04-19";
```

Comment:
They had a beta a few months earlier than this but since it was just a beta instead of a stable released I just tested latest revision instead.